site/en/blog/renderingng/index.md

```diff
@@ -0,0 +1,554 @@
+and scaling down—maximizing efficiency and reducing demand on the system when needed.
+
+{% Img src="image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/Hdfm4k1cSwfJpffgx3eH.jpg",
+alt="", width="800", height="533" %}
+
+To get there, we needed to make maximum use of caching,
+performance isolation and GPU hardware acceleration.
+Let's consider each in turn. And to make it concrete,
+let's think about how each of them contributes to the performance of one extremely important interaction on web pages: scrolling.
+
+### Caching
+
+In a dynamic, interactive UI platform such as the web,
+caching is the single most important way to dramatically improve performance.
+The most well-known kind of caching in a browser is the HTTP cache,
+but rendering also has many caches.
+The most important cache for scrolling is cached GPU textures and display lists,
+which allow scrolling to be extremely fast while minimizing battery drain and working well across a variety of devices.
+
+Caching helps battery life and animation frame rate for scrolling,
+but even more important is that it unblocks performance isolation from the main thread.
+
+### Performance isolation
+
+On modern desktop computers,
+you never have to worry about background applications slowing down the one you're working in.
+That's because of preemptive multitasking,
+which is in turn a form of performance isolation:
+making sure independent tasks don't slow each other down.
+
+On the web, the best example of performance isolation is scrolling.
+Even on websites that have lots of slow JavaScript,
+scrolling can be very smooth,
+because it runs on a different thread that doesn't have to depend on the JavaScript and layout thread.
+We put a ton of effort into RenderingNG to make sure that every possible scroll is threaded,
+through caching that goes well beyond just a display list to more complex situations.
+Examples include code to represent fixed- and sticky-positioned elements,
+passive event listeners, and high-quality text rendering.
+
+{% Img src="image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/ooz0P0bUN3x69HDzJWaW.jpg",
+alt="Sketch shows that with RenderingNG performance stays solid.",
+width="800", height="559" %}
+
+### GPU acceleration
+
+A GPU makes generating pixels and drawing to the screen dramatically faster—
+in many cases, every pixel can be drawn in parallel with every other pixel,
+resulting in an enormous speed increase.
+A key component of RenderingNG is GPU raster/draw everywhere.
+This uses the GPU on all platforms, and all devices, to hyper-accelerate the rendering and animating of web content.
+This is especially important on low-end devices or very high-end ones,
+which often have a much more capable GPU than other parts of the device.
+
+
+{% Img src="image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/2kly2kdoelhTeDqUHEPn.jpg",
+alt="Sketch shows that with RenderingNG performance does not degrade so much.",
+width="800", height="559" %}
+
+## Extensibility: the right tools for the job
```

Comment:
```suggestion
## Extensibility: The right tools for the job
```

Reply:
Capitalize the first word after a colon in subheadings https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings